### DISCUSSION TITLE
Minimizing Padding/Space between header and SearchResult WebPart

### DISCUSSION THREAD

**Discussion by ynck-codes** [microsoft-search/pnp-modern-search#3405] [Q&A] [answered]
I may oversee something but is there any chance to get rid of the marked space in the screenshot?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/102553711/c89e7c62-9d7b-4f65-a38f-98e46ccf2a68)

**Comment by wobba** [comment, marked as answer]
Pick the people template. Click {} to edit it, and remove the section below, or any other section you want :)

```
template--sort (edit)
```

**Reply by ynck-codes** [reply]
Thanks @wobba. Deleting template--sort instead of template--header will do the magic at least for me.

**Reply by wobba** [reply]
You are correct.....typo from me 🙂 May look at the CSS or default template and omit the part if it's not rendering. But al least you have a work-around.

**Comment by ynck-codes** [comment]
@wobba the following might be a whole new issue but is connected to the explanation above. 
You have mentioned that rendering might be a little bumpy. 

Indeed, as soon as I delete the template--sort div, with "persona card show on hover" activated beforehand, the whole persona list won't render. 

Any clue why that happens?

**Reply by wobba** [reply]
No idea, and log a new issue. And you might want to use the native SharePoint people card option added in v4.10.1 instead.
